Automattic/vip-cli
# Reverted breaking change in PR #1049

## Description

This PR reverts the code on PR #1049 while we investigate it further.

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Run `npm install`
1. Run `npm run build`
1. Run `node dist/bin/vip-app list` runs fine.
